assets/stylesheets/gu-sass/layout.scss

```diff
@@ -8,6 +8,7 @@
 $gu-h-spacing: 20px;
 $gu-v-spacing: 12px;
 $gu-col-width: 60px;
+$gu-input-height: $gu-v-spacing * 3.5;
```

Comment:
Is this different from `$cta-height` underneath?

Reply:
funny story that! `$cta-height` was _slightly wrong_ all along, (42px vs 44px) i will clean it up!